spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -2872,6 +2887,27 @@ def run_pygments_highlighter(self):
         if isinstance(self.highlighter, sh.PygmentsSH):
             self.highlighter.make_charlist()
 
+    def left_line(self, selection_position):
+        """If we left a line with only spaces, remove them (W293)"""
+        cursor = self.textCursor()
+        current_pos = cursor.position()
+        # Chech if still on the line
+        if min(selection_position) <= current_pos and \
+                max(selection_position) >= current_pos:

```

Comment:
Please enclose this condition in parenthesis instead of using `\`, like this:

```python
if (min(selection_position) <= current_pos and
        max(selection_position) >= current_pos):
```